home/wagtail_hooks.py

```diff
@@ -200,6 +207,48 @@ def page(self, obj):
 
     page.short_description = "Page Slugs"
 
+    def time(self, obj):
+        if obj.pages:
+            return [
+                (
+                    f"{p.value['time']} {p.value['unit']}"
+                    if p.value and "time" in p.value
```

Comment:
Should we maybe have unit be a separate field, or add it to this if statement? Otherwise it is possible to have time but no unit, since they're both optional fields.

Reply:
Good point. I've made it a separate field.